server/src/test/java/com/linecorp/centraldogma/server/metadata/LoginStringifierTest.java

```diff
@@ -0,0 +1,119 @@
+ * under the License.
+ */
+package com.linecorp.centraldogma.server.metadata;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.function.Consumer;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+import com.google.common.collect.Maps;
+
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.logback.RequestContextExportingAppender;
+import com.linecorp.armeria.common.util.SafeCloseable;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.logging.AccessLogWriter;
+import com.linecorp.centraldogma.common.Author;
+import com.linecorp.centraldogma.server.internal.admin.auth.AuthUtil;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+
+class LoginStringifierTest {
+
+    private static final Logger rootLogger =
+            (Logger) LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
+    private static final LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
+
+    @BeforeEach
+    void beforeEach() throws Exception {
+        rootLogger.getLoggerContext().getStatusManager().clear();
+        MDC.clear();
+
+        final JoranConfigurator configurator = new JoranConfigurator();
+        configurator.setContext(context);
+        context.reset();
+        configurator.doConfigure(getClass().getResource("testLoginStringifier.xml"));
+    }
+
+    @AfterEach
+    void afterEach() throws Exception {
+        final JoranConfigurator configurator = new JoranConfigurator();
+        configurator.setContext(context);
+        context.reset();
+
+        configurator.doConfigure(getClass().getResource("/logback-test.xml"));
+    }
+
+    @Test
+    void requestContextExportingAppender() {
+        final Logger logger = (Logger) LoggerFactory.getLogger("RCEA");
+        final RequestContextExportingAppender rcea =
+                (RequestContextExportingAppender) logger.getAppender("RCEA");
+
+        final ListAppender<ILoggingEvent> la = new ListAppender<>();
+        rcea.addAppender(la);
+        rcea.start();
+        la.start();
+        final ServiceRequestContext ctx = contextWithToken(sb -> {});
+        try (SafeCloseable ignored = ctx.push()) {
+            logger.trace("log");
+            assertThat(la.list.remove(0).getMDCPropertyMap()).containsExactly(
+                    Maps.immutableEntry("attrs.app_id", "foo"));
```

Comment:
nit; can use assertj's `entry`
```suggestion
                    entry("attrs.app_id", "foo"));
```